sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/CassandraKeyspaceGetResults.cs

```diff
@@ -42,8 +42,8 @@ public CassandraKeyspaceGetResults()
         /// <param name="type">The type of Azure resource.</param>
         /// <param name="location">The location of the resource group to which
         /// the resource belongs.</param>
-        public CassandraKeyspaceGetResults(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), CassandraKeyspaceGetPropertiesResource resource = default(CassandraKeyspaceGetPropertiesResource), CassandraKeyspaceGetPropertiesOptions options = default(CassandraKeyspaceGetPropertiesOptions))
-            : base(id, name, type, location, tags)
+        public CassandraKeyspaceGetResults(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), ManagedServiceIdentity identity = default(ManagedServiceIdentity), CassandraKeyspaceGetPropertiesResource resource = default(CassandraKeyspaceGetPropertiesResource), CassandraKeyspaceGetPropertiesOptions options = default(CassandraKeyspaceGetPropertiesOptions))
```

Comment:
Please confirm that all of the breaking changes like this one have been introduced in previous preview version `3.2.0-preview`(Swagger version should be 2021-07-01-preview). If so, I think we should update the version to `3.4.0-preview`.

Reply:
I also tried updating Assembly.cs to 3.4.0.0-preview, but it is complaining as invalid. Should i update it as 3.4.0.0 ?